README.rst

```diff
@@ -52,26 +52,31 @@ Using poetry
 
 .. code-block:: bash
 
-    poetry add sphinx-needs
+    poetry add sphinx-needs[all]
+    # Lighter install without matplotlib = no pie charts
+    poetry add sphinxcontrib-needs
```

Comment:
Why does it still say sphinxcontrib-needs here? Perhaps a merge error?

Reply:
Fixed the sphinxcontrib-needs name, but if I remove the comment we have these 2 without explanation?
```rst
    poetry add sphinx-needs[all]
    poetry add sphinx-needs
```
That felt weird so I added that comment. Happy to remove, mostly felt the unexplained `[all]` would add confusion.